src/test/uk/gov/hmrc/mongoFeatureToggles/services/FeatureFlagServiceWithCacheSpec.scala

```diff
@@ -162,4 +162,17 @@ class FeatureFlagServiceWithCacheSpec extends BaseSpec {
       verify(mockFeatureFlagRepository, times(1)).getAllFeatureFlags
     }
   }
+
+  "getAsEitherT" must {
+    "do the same as get but return an EitherT" in {
+      val expectedFeatureFlag = FeatureFlag(TestToggleA, math.random() < 0.5)
```

Comment:
If you don't care if it's true of false, this can be explicitly set to either.

Reply:
The test does not depend on the expected value itself. it only check that it is consistent. The random value is an attempt to show this.